mpas_analysis/shared/plot/plotting.py

```diff
@@ -27,6 +28,187 @@
 
 from ..constants import constants
 
+
+def nino34_spectra_plot(config, f, ninoSpectra, confidence95, confidence99,
+			redNoiseSpectra, title, xlabel, ylabel,
+                        fileout, linewidths, titleFontSize=None,
+			figsize=(15, 6), dpi=300):
+    """
+    Plots the nino34 time series and power spectra in an image file
+    Parameters
+    ----------
+    config : instance of ConfigParser
+        the configuration, containing a [plot] section with options that
+        control plotting
+
+	f : frequencies to plot on x-axis
+
+    ninoSpectra : nino34 power spectra
+
+    confidence95 : 95% confidence level based on chi squared test
+
+    confidence99 : 99% confidence level based on chi squared test
+
+    redNoiseSpectra : red noise fit to the ninoSpectra
+
+    title : str
+        the title of the plot
+
+    xlabelTimeSeries, ylabelTimeSeries, xlabelSpectra, ylabelSpectra : str
+        axis labels
+
+    fileout : str
+        the file name to be written
+
+    linewidths : control line width
+
+    titleFontSize : int, optional
+        the size of the title font
+
+    figsize : tuple of float, optional
+        the size of the figure in inches
+
+    dpi : int, optional
+        the number of dots per inch of the figure
+
+    Author: Luke Van Roekel
+    Last Modified: 03/21/2017
+    """
+
+    plt.plot(f[2:-3], ninoSpectra[2:-3], 'k', linewidth=linewidths)
```

Comment:
An idea-- maybe just write something like this so that you can easily change the limits later if needed:
```python
# comment on why you are plotting the [2:-3] slice
def plot_slice(x, y, *wargs, slice=slice(2,-3),  **kwargs):
    plt.plot(x[slice], y[slice], *wargs, **kwargs)

plot_slice(f, ninoSpectra, 'k', linewidth=linewidths)
...
```